packages/ui/src/molecules/Incentives/Incentives.test.tsx

```diff
@@ -0,0 +1,61 @@
+import { render, cleanup } from '@testing-library/react'
+import { axe } from 'jest-axe'
+import React from 'react'
+
+import Incentives from './Incentives'
+import IncentivesItem from './IncentivesItem'
+
+const TestIncentives = () => {
+  return (
+    <Incentives>
+      <IncentivesItem>
+        <span>Item title</span>
+        <span>Item description</span>
+      </IncentivesItem>
+      <IncentivesItem>
+        <span>Another item title</span>
+        <span>Another item description</span>
+      </IncentivesItem>
+    </Incentives>
+  )
+}
+
+describe('Incentives', () => {
+  let incentives: HTMLElement
+  let incentivesItems: HTMLElement[]
+  let incentivesPure: Element | null
+
+  beforeEach(() => {
+    const { getByTestId, getAllByTestId, container } = render(
+      <TestIncentives />
+    )
+
+    incentives = getByTestId('store-incentives')
+    incentivesItems = getAllByTestId('store-incentives-item')
+    incentivesPure = container.querySelector('section')
```

Comment:
Does getByRole('section') work?

Reply:
@igorbrasileiro no :( I tried to use this approach first, but the `getByRole` can't find the `section` role by the semantic tag `<section>`

Reply:
Ok ok, np!